extensions/easyvariable/src/components/FormatList.tsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from "react";
+import { ActionPanel, Action, Icon, List, Clipboard, Keyboard } from "@raycast/api";
+import { formatCamelCase } from "../utils/formatters/camelCase";
+import { formatPascalCase } from "../utils/formatters/pascalCase";
+import { formatSnakeCase } from "../utils/formatters/snakeCase";
+import { formatKebabCase } from "../utils/formatters/kebabCase";
+import { formatConstant } from "../utils/formatters/constant";
+
+interface FormatListProps {
+  text: string;
+}
+
+export function FormatList({ text }: FormatListProps) {
+  const formats = [
+    { name: "camelCase", format: formatCamelCase, example: "userName", alias: "xt" },
+    { name: "PascalCase", format: formatPascalCase, example: "UserName", alias: "dt" },
+    { name: "snake_case", format: formatSnakeCase, example: "user_name", alias: "xh" },
+    { name: "kebab-case", format: formatKebabCase, example: "user-name", alias: "hx" },
+    { name: "CONSTANT_CASE", format: formatConstant, example: "USER_NAME", alias: "cl" },
```

Comment:
What does the `xt dt xh hx cl` mean? I don't think they are good aliases for non-Chinese users.

Feel free to remove them since Raycast already supported searching for terms in titles and subtitles.